src/uu/tail/src/platform/unix.rs

```diff
@@ -45,7 +45,9 @@ pub fn supports_pid_checks(pid: self::Pid) -> bool {
 fn get_errno() -> i32 {
     Error::last_os_error().raw_os_error().unwrap()
 }
+
 #[inline]
+#[allow(dead_code)]
```

Comment:
Please don't do this. These annotations are hard to remove again because they don't report when they are not necessary anymore.

Reply:
Removing this function entirely then (together with platform::windows::stdin_is_pipe_or_fifo)? I haven't found a usage for it, but didn't want to remove it completely right away.

Reply:
If it's not not used it can indeed be removed

Reply:
Ok